internal/pkg/cli/secret_init.go

```diff
@@ -146,33 +170,44 @@ func (o *secretInitOpts) Ask() error {
 	return nil
 }
 
-// Execute creates the secrets.
+// Execute creates or updates the secrets.
 func (o *secretInitOpts) Execute() error {
-	secretName := o.name
-	secretValues := o.values
-
 	if o.inputFilePath != "" {
-		var err error
-		secretName, secretValues, err = o.parseFile()
+		secrets, err := o.parseSecretsInputFile()
 		if err != nil {
 			return err
 		}
+		for secretName, secretValues := range secrets {
+			o.putSecret(secretName, secretValues)
+		}
+		return nil
 	}
 
-	return o.putSecrets(secretName, secretValues)
+	o.putSecret(o.name, o.values)
+	return nil
 }
 
-func (o *secretInitOpts) putSecrets(secretName string, values map[string]string) error {
+func (o *secretInitOpts) putSecret(secretName string, values map[string]string) {
+	spinner := termprogress.NewSpinner(log.DiagnosticWriter)
+
+	spinner.Start("start creating secret")
```

Comment:
```suggestion
	spinner.Start(fmt.Sprintf("Start creating secret %s.", secretName))
```

Reply:
What do you think of removing spinner? I am thinking I should remove them since `PutSecret` is usually a very quick call that's returned under a second.

Reply:
yeah that's my concern as well

Reply:
Can you include an example output for this feature? like what the output would be if they are using file input?

Reply:
hahaha I will just go ahead and remove it then. They are not very helpful in this context.

Reply:
I updated the log messages and errors, as well as the example ⬇️